smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpBindingProtocolGenerator.java

```diff
@@ -633,6 +647,25 @@ private void generateErrorDeserializer(GenerationContext context, StructureShape
         writer.write("");
     }
 
+    private List<HttpBinding> readErrorResponseBody(
+            GenerationContext context,
+            Shape error,
+            HttpBindingIndex bindingIndex
+    ) {
+        TypeScriptWriter writer = context.getWriter();
+        if (isErrorCodeInBody) {
+            // Body is already parsed in error dispatcher, simply assign body to data.
+            writer.write("const data: any = output.body;");
+            List<HttpBinding> responseBindings = bindingIndex.getResponseBindings(error).values()
```

Comment:
This will contain all response bindings for the `error` shape, use `Location.DOCUMENT` to refine this.

Reply:
Will update this.